awsconfigs/apps/pipeline/kustomization.yaml

```diff
@@ -27,6 +27,7 @@ generatorOptions:
 patchesStrategicMerge:
 - ./rds/disable-mysql-pv-claim.yaml
 - ./rds/disable-default-secret.yaml
+- ./s3/remove-minio-server-resources.yaml
```

Comment:
can we use disable instead of remove for consistency

Reply:
renamed